packages/mds-db/policies.ts

```diff
@@ -179,19 +179,23 @@ export async function publishPolicy(policy_id: UUID) {
         geographies.push(geography_id)
       })
     })
-    await Promise.all(
-      geographies.map(geography_id => {
-        log.info('publishing geography', geography_id)
-        return publishGeography({ geography_id, publish_date })
-      })
-    )
-    await Promise.all(
-      geographies.map(geography_id => {
-        const ispublished = isGeographyPublished(geography_id)
-        log.info('published geography', geography_id, ispublished)
+
+    const unpublishedGeoIDs = await Promise.all(
+      geographies.map(async geography_id => {
+        const isPublished = await isGeographyPublished(geography_id)
+        if (!isPublished) {
+          return geography_id
+        }
+        return null
       })
     )
 
+    unpublishedGeoIDs.forEach(id => {
```

Comment:
You could potentially use `filterEmpty()` here to save yourself an `if` statement